docs/relations_guide.md

```diff
@@ -1,103 +1,123 @@
 
-e.g. 'hilus cell of ovary' **'capable of'** *some* 'androgen secretion'
+### Recording anatomical location (cells in immaterial spaces)
 
-## Recording developmental lineage
+To record the location of a cell in an anatomical space (e.g., a sinus), [**'located in'**](http://purl.obolibrary.org/obo/RO_0001025) is used. For example, [lymph node marginal reticular cell](http://purl.obolibrary.org/obo/CL_0009103) has the following subclass:
 
-We record developmental lineage relationships between cell types using **develops from**, or where we are sure there are not intermediates between the related cells, by using **'develops directly from'**
 
-For example:
+[**'located in'**](http://purl.obolibrary.org/obo/RO_0001025) *some* ['subcapsular sinus of lymph node'](http://purl.obolibrary.org/obo/UBERON_0005463)
 
-'leukocyte' **develops from** *some* 'hematopoietic stem cell'
 
-## Relations for neurons
+### Recording function
 
-### Synaptic connectivity
+Cellular function is recorded by linking GO biological process terms with the object property [**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215). 
 
-To record neuron to neuron or motor neuron -> target cell connectivity use.  Use these relations sparingly where connectivity is key to definition, e.g. motor neuron types defined by the type of muscle cell they synapse to.
+For example, ['hilus cell of ovary'](http://purl.obolibrary.org/obo/CL_0002095) has the following subclass:
 
-**synapsed to** - preferred direction to record, as it fits with 
+[**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215) *some* ['androgen secretion'](http://purl.obolibrary.org/obo/GO_0035935)
 
-**synapsed_by** - Useful in cases where all X synapsed by some Y but there reverse is not true
 
-To record connection between a neuron and a region it innervates we have a number of relations, all sub properties of overlaps
+### Recording developmental lineage
 
-![image](https://user-images.githubusercontent.com/112839/94337631-e0a83300-ffe3-11ea-8f13-ac8a484a5fb3.png)
+Developmental lineage is recorded between cell types with the object property [**develops from**](http://purl.obolibrary.org/obo/RO_0002202), or in the case where there are no intermediates between the cells, [**'directly develops from'**](http://purl.obolibrary.org/obo/RO_0002207).
 
-Historically we have used **has(pre/post)synaptic terminal in** to record this. However, these relations are defined as being true when a single synapse is present in a region.  As these relations are also used with data, this turns out to be too sensitive to biological and experimental noise.  We therefore now prefer the more specific relations for defining classes - defined in terms of functionally significant synaptic inputs/output in a region.
+For example, ['leukocyte'](http://purl.obolibrary.org/obo/CL_0000738) has the following subclass:
 
```

Comment:
Important context cut here.  If not clear enough we should find a way to bring back in clearer form.